korangar/src/graphics/passes/directional_shadow/shader/entity.wgsl

```diff
@@ -60,29 +60,30 @@ fn vs_main(
         output.texture_coordinates.x = 1.0 - output.texture_coordinates.x;
     }
 
+    let MAX_X = 500.0;
```

Comment:
Why do we use 500 here?

Reply:
It the sprite max size will change the naming

Reply:
I am using the wrong value to calculate the proportion, I will fix it.